corehq/apps/settings/views.py

```diff
@@ -252,7 +252,8 @@ def post(self, request, *args, **kwargs):
             self.settings_form.update_user()
 
             res = redirect(reverse(MyAccountSettingsView.urlname))
-            res.set_cookie(settings.LANGUAGE_COOKIE_NAME, self.request.couch_user.language)
+            res.set_cookie(settings.LANGUAGE_COOKIE_NAME, self.request.couch_user.language,
+                           secure=settings.SECURE_COOKIE)
```

Comment:
@gherceg this should be `SECURE_COOKIES`: https://sentry.io/organizations/dimagi/issues/3494689565/

Reply:
🤦🏻 thanks for catching this @snopoke. Resolved in the bbc393a5d5c3eb6f234bf6744521a4b79e599b7f